src/Symfony/Component/Validator/Mapping/Loader/PropertyInfoLoader.php

```diff
@@ -47,7 +47,7 @@ public function __construct(PropertyListExtractorInterface $listExtractor, Prope
     public function loadClassMetadata(ClassMetadata $metadata)
     {
         $className = $metadata->getClassName();
-        if (null !== $this->classValidatorRegexp && !preg_match($this->classValidatorRegexp, $className)) {
```

Comment:
If null is passed, it means that it should be always executed IIRC.

Reply:
I don't think it should be the case, especially with: https://github.com/symfony/symfony/pull/31749/files#diff-0bd78268e7d70f726415dc3175eaa595R76